demisto_sdk/common/hook_validations/integration.py

```diff
@@ -51,7 +51,8 @@ def is_valid_file(self, validate_rn=True):
             self.is_insecure_configured_correctly(),
             self.is_valid_category(),
             self.is_id_equals_name(),
-            self.is_docker_image_valid()
+            self.is_docker_image_valid(),
+            self.is_valid_feed()
```

Comment:
next addition wont show as two line changed
```suggestion
            self.is_valid_feed(),
```